src/terrain/routes.clj

```diff
@@ -247,6 +248,7 @@
                                      {:name "admin-reference-genomes", :description "Admin Reference Genome Endpoints"}
                                      {:name "admin-requests", :description "Admin Request Endpoints"}
                                      {:name "admin-settings", :description "Admin Setting Endpoints"}
+                                     {:name "admin-token", :description "Admin OAuth Tokens"}
```

Comment:
I briefly considered changing the wording to `Admin OAuth and OIDC Tokens`. Ultimately, however, OIDC tokens _are_ OAuth tokens. I figured this was close enough.